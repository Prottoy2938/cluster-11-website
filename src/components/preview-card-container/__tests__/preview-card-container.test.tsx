import React from "react";
import { render, cleanup } from "@testing-library/react";
import PreviewCardContainer from "../preview-card-container";
afterEach(cleanup);

test("should take and match snapshot", () => {
  const { asFragment } = render(
    <PreviewCardContainer APIEndPoints={["placeholder", "text", "here"]} />
  );
  expect(
    asFragment(
      <PreviewCardContainer APIEndPoints={["placeholder", "text", "here"]} />
    )
  ).toMatchSnapshot();
});
