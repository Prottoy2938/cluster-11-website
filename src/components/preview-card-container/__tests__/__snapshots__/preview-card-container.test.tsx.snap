// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take and match snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1"
    />
  </div>
</DocumentFragment>
`;
